### PR TITLE
Cleanups and some fixes

### PR DESCRIPTION
**Pull request by Quix0r**
This PR fixes CHMOD on non-executable files + an `E_NOTICE` in `boot.php`. It also renamed `.htconfig.php` as ignored files should never be committed (and this one contains sensitive informations). If only the line in `.gitignore` is removed, sensitive information can be committed and consequently published.

I also gave all JavaScript files a separate folder so "include" (if no longer needed) can be secured by .htaccess file (`Deny from all` should be okay).

The JavaScript library "Raphael" (or so?) is now residing in an own sub folder.

Most changes on those 185 files are CHMOD, don't wonder about it. :-)

### CODE REVIEW

File: view/head.tpl [hunk 1]
```diff
@@ -3,13 +3,14 @@
...
+//-->
 <![endif]-->
 
-<link rel="shortcut icon" href="$baseurl/images/friendica-32.png" />
+<link rel="shortcut icon" href="images/friendica-32.png" />
 
-<script type="text/javascript" src="$baseurl/include/jquery.js" ></script>
-<script type="text/javascript" src="$baseurl/include/main.js" ></script>
+<script type="text/javascript" src="js/jquery.js" ></script>
+<script type="text/javascript" src="js/main.js" ></script>
```
Comment (Hypolite) [inline view/head.tpl]: Can you please leave the $baseurl for eventual subfolder urls (like `/servers`)?

File: view/#head.tpl# [hunk 2]
```diff
@@ -1,10 +0,0 @@
-<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
-<base href="$baseurl" />
-<link rel="stylesheet" type="text/css" href="$stylesheet" media="all" />
-
-<!--[if IE]>
-<script type="text/javascript" src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-<![endif]-->
-<script type="text/javascript" src="$baseurl/include/jquery.js" ></script>
-<script type="text/javascript" src="$baseurl/include/main.js" ></script>
-
```
Comment (Hypolite) [inline view/#head.tpl#]: Wait, why did you remove this file?

File: view/head.tpl [hunk 3]
```diff
@@ -3,13 +3,14 @@
...
+//-->
 <![endif]-->
 
-<link rel="shortcut icon" href="$baseurl/images/friendica-32.png" />
+<link rel="shortcut icon" href="images/friendica-32.png" />
 
-<script type="text/javascript" src="$baseurl/include/jquery.js" ></script>
-<script type="text/javascript" src="$baseurl/include/main.js" ></script>
+<script type="text/javascript" src="js/jquery.js?v=1.4.2" ></script>
+<script type="text/javascript" src="js/main.js" ></script>
```
Comment (Hypolite) [inline view/head.tpl]: Can you please leave the $baseurl for eventual subfolder urls (like `/servers`)?

File: .htconfig.php-dist [hunk 4]
```diff
@@ -75,4 +75,4 @@
   //At which health value should we start removing profiles?
   'remove_profile_health_threshold' => -60
   
-);
\ No newline at end of file
+);
```
Comment (tobiasd) [inline .htconfig.php-dist]: The "dist" file is called `htconfig.php` without the dot in Friendica. I think it should be the same here.

**Comment by Beanow** [comment]
Be sure to have a look at https://github.com/Beanow/dir/tree/feature/redesign-prototype as well. There I had added `/assets/js` as I also added other static files like fonts and style sheets to this folder.

File: .htconfig.php-dist (hunk 4 above)
Comment (Quix0r) [inline .htconfig.php-dist]: Okay, will rename it when I'm home.

File: view/head.tpl (hunk 3 above)
Comment (Quix0r) [inline view/head.tpl]: There is  `<base href="$baseurl" />` in line 2 which sets the base URL for all other URLs. I can test it at home where it will definedly be in a sub folder.

**Comment by Quix0r** [comment]
@Beanow so I move them to `/asset/js/` and later try to merge your changes into mine?

File: view/head.tpl (hunk 3 above)
Comment (Hypolite) [inline view/head.tpl]: That would be great. I don't use `<base>` because I prefer explicit absolute paths to implicit prepending, but I reckon it would work.

Comment (Quix0r) [inline view/head.tpl]: Okay, I didn't know that. I saw the same in `Friendica` and thought this might be the same here, too.

Comment (annando) [inline view/head.tpl]: I guess this are two different philosophies :-) I'm more with the ````<base>```` stuff.

**Comment by Hypolite** [comment]
This PR has been included in the redesign-prototype branch I'm running right now.
